Mf2/Parser.php

```diff
@@ -1506,6 +1506,8 @@ public function backcompat(DOMElement $el, $context = '', $isParentMf2 = false)
 			$mf1Classes = array_intersect($classes, array_keys($this->classicRootMap));
 		}
 
+		$el_has_mf2 = $this->hasRootMf2($el);
```

Comment:
the code mainly uses camel case vars, shouldn't we keep that, to be consistent?

Reply:
Good point. I'll revise.